docs/amber_script.md

```diff
@@ -207,8 +203,6 @@ The following commands are all specified within the `PIPELINE` command.
  * uniform
  * storage
 
-TODO(dsinclair): Reserve `input` as a buffer type for input attachments.
```

Comment:
Colour attachments aren't a buffer type, they're done with ATTACH now.

Reply:
Input attachments are not colour attachments.  They are another kind of attachment with their own index space.  See InputAttachmentIndex decoration in SPIR-V.

But point taken, the syntax should be something like:

    BIND BUFFER  myInputAttachment AS input LOCATION 0

(reusing LOCATION here, but could just as well be INDEX.)